File: Beginner/1015 - Distance Between Two Points/1015.c
#include <stdio.h>
#include <math.h>

int main(){

    double x1, y1, x2, y2, distancia, m1, m2;
    scanf("%lf %lf", &x1, &y1);
    scanf("%lf %lf", &x2, &y2);
    m1 = pow((x2-x1), 2);
    m2 = pow((y2-y1), 2);
    distancia = sqrt(m1 + m2);
    printf("%.4lf\n", distancia);
    return 0;
}

File: Beginner/1015 - Distance Between Two Points/1015.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        String[] input;
        input = Console.ReadLine().Trim().Split(' ');
        double x1 = double.Parse(input[0]);
        double y1 = double.Parse(input[1]);

        input = Console.ReadLine().Trim().Split(' ');
        double x2 = double.Parse(input[0]);
        double y2 = double.Parse(input[1]);

        double D = Math.Sqrt(Math.Pow((x2 - x1), 2) + Math.Pow((y2 - y1), 2));
        Console.WriteLine($"{D:0.0000}");
    }
}